### PR TITLE
[Feature] Add cancel button for embargoed registrations. [OSF-6145]

### PR DESCRIPTION
**Pull request by TomHeatwole**
## Purpose

Allow all registrations to be cancelled including ones with an embargo.

## Changes

- Added the button to the project header page
- Added the disapproval link to the project registration file

Before:

After:

## Side effects
N/A

## Ticket

https://openscience.atlassian.net/browse/OSF-6145

### CODE REVIEW

**Comment by brianjgeiger** [comment]
Looks good code-wise and the screenshots look right. After product demo it should be good to go. 🎉 🐧 🎉

File: website/templates/project/project_header.mako [hunk 1]
```diff
@@ -139,7 +139,18 @@
...
 
         % if node['is_pending_embargo']:
-            <div class="alert alert-info">This ${node['node_type']} is currently pending registration, awaiting approval from project administrators. This registration will be final and enter the embargo period when all project administrators approve the registration or 48 hours pass, whichever comes first. The embargo will keep the registration private until the embargo period ends.</div>
+            <div
+                class="alert alert-info">This ${node['node_type']} is currently pending registration, awaiting approval from project administrators. This registration will be final and enter the embargo period when all project administrators approve the registration or 48 hours pass, whichever comes first. The embargo will keep the registration private until the embargo period ends.
+                % if 'admin' in user['permissions']:
+                        <div>
+                            <br>
+                            <button type="button" id="registrationCancelButton" class="btn btn-danger" data-toggle="modal" data-target="#registrationCancel">
+                                Cancel Registration
```
Comment (sloria) [inline website/templates/project/project_header.mako]: As per osf-style, use sentence-case for button text. 

If this needs to be changed on any other "Cancel Registration" buttons, go ahead and change it in those places, too.

**Comment by sloria** [comment]
Just a minor change to make, and this should be good to merge.

File: website/templates/project/project_header.mako [hunk 2]
```diff
@@ -144,7 +144,7 @@
                 % if 'admin' in user['permissions']:
                         <div>
                             <br>
-                            <button type="button" id="registrationCancelButton" class="btn btn-danger" data-toggle="modal" data-target="#registrationCancel">
+                            <button type="button" id="registrationCancelButton" class="btn btn-danger" data-toggle="modal" data-target="#registrationCancel" text-case="sentence">
```
Comment (sloria) [inline website/templates/project/project_header.mako]: Why not just change the text to "Cancel registration"?

Comment (sloria) [inline website/templates/project/project_header.mako]: I'll make this change when I merge.

**Pull request merged**